src/Playroom/CodeEditor/CodeEditor.css.ts

```diff
@@ -23,6 +23,9 @@ export const errorMarker = style([
     opacity: 0,
   }),
   {
+    ':hover': {
+      cursor: 'pointer',
+    },
```

Comment:
Should `cursor: 'pointer'` be used when there isn't really a mouse action to be performed? Maybe a hover style would be more appropriate?